app_integrations/apps/app_base.py

```diff
@@ -222,6 +222,24 @@ def _make_request(self, full_url, headers, params):
 
         return response.json()
 
+    def _make_post_request(self, full_url, headers, json):
```

Comment:
Can we choose a different name from `json` for the variable, since that conflicts with the `json` package?